restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCConnection.java

```diff
@@ -560,10 +563,19 @@ void initializeWebrtc(boolean videoEnabled)
         for (String permission : MANDATORY_PERMISSIONS) {
             if (context.checkCallingOrSelfPermission(permission) != PackageManager.PERMISSION_GRANTED) {
                 logAndToast("Permission " + permission + " is not granted");
-                // TODO: return error to RCConnection listener
+                listener.onPermissionNotGranted(this, permission);
```

Comment:
Yeah, sounds like a plan, let's get done with the SDK before update the App

On Thu, May 12, 2016 at 1:41 PM, mehturt <notifications@github.com> wrote:

> In
> restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCConnection.java
> <https://github.com/RestComm/restcomm-android-sdk/pull/283#discussion_r62999416>
> :
>
> > @@ -560,10 +563,19 @@ void initializeWebrtc(boolean videoEnabled)
> >          for (String permission : MANDATORY_PERMISSIONS) {
> >              if (context.checkCallingOrSelfPermission(permission) != PackageManager.PERMISSION_GRANTED) {
> >                  logAndToast("Permission " + permission + " is not granted");
> > -                // TODO: return error to RCConnection listener
> > +                listener.onPermissionNotGranted(this, permission);
>
> Yes. I also planned to update the example application so that it compiles,
> but perhaps only after the SDK is reviewed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/RestComm/restcomm-android-sdk/pull/283/files/7e91af930a845199f3ed94e753307d408df57d0f#r62999416>
>



-- 
Antonis Tsakiridis
Lead Developer, Mobile SDKs at TeleStax
antonis.tsakiridis@telestax.com
http://www.telestax.com